### ISSUE TITLE
Triangle tag inside attribute after new line causes parse error

### ISSUE THREAD

**Issue by Danon** [valentinnodan/mdx-intellij-plugin#9]
You can have regular tags inside regular strings or template strings:
```mdx
<Component attr={'<'}/>
<Component attr={`<`}/>
<Component attr="<"/>
```

but if you move the attribute to the next line, there is parse error
```mdx
<Component 
    attr={`<`}/>
```

**Comment by Danon** [comment]
Actually, even code like this causes parse error:
```mdx
<div>
  {`<test>`}
</div>
```

**Comment by vladsholokhov** [comment]
@Danon 
Hi, Could you share the exact file/ screenshot with the issue?

**Comment by Danon** [comment]
Sure, I'm using `1.0.3-SNAPSHOT`.

Here's the parse error: JSX tag, newline, and attribute with a template tag that holds `<` character.
![image](https://user-images.githubusercontent.com/13367735/109635465-b33f5280-7b4a-11eb-98a2-ad344d5ef1eb.png)

If I remove the new line, or I remove the `<` character, or I replace template string with a regular `"` or `'` quote string, everything works ok. 
![image](https://user-images.githubusercontent.com/13367735/109635545-cce09a00-7b4a-11eb-934a-bfae0b22b157.png)
![image](https://user-images.githubusercontent.com/13367735/109635556-d10cb780-7b4a-11eb-9932-2030fb2e7736.png)


But In my case I need both the new line, and `<` and the template string.

**Comment by vladsholokhov** [comment]
Thank you!

Moving `/>` to the next line fixes the issue as well.
![image](https://user-images.githubusercontent.com/4963635/109645129-77ff4c80-7b67-11eb-9cca-12d7f01002ae.png)

Original issue:
```### Test

<CodeTabs
    tregx={``}
    php={`<`}/>

Of course, `first()` callback will asd qwe

:::note
```

```
WebStorm 2021.1 Beta
Build #WS-211.6305.4, built on March 2, 2021
```

**Comment by Danon** [comment]
@vladsholokhov Oh, that's good news, at least I can correct my MDX files now. But do you think we can fix the issue anyway?

**Comment by vladsholokhov** [comment]
сс: @anstarovoyt

**Comment by Danon** [comment]
@vladsholokhov Yea, but using `<>` in the string still breaks it, even with the enter.

```jsx
<CodeTabs
    tregx={`<>`}
    php={`<`}
/>
```

**Comment by Danon** [comment]
@vladsholokhov So can we do anything about it?

**Comment by Danon** [comment]
@vladsholokhov Do you think we can start some progress here?

**Comment by Danon** [comment]
@vladsholokhov Can you point me in the right direction, so maybe I can fix the bug myself?

**Comment by anstarovoyt** [comment]
@Danon Hi! I've published the "1.2.211" version with the fix, please give it a try.

**Comment by anstarovoyt** [comment]
Unfortunately, the problem is rather general, so if you see something similar please let us know.

**Comment by Danon** [comment]
Sure, thanks! I'll update the plugin and verify the fix!

śr., 12 maj 2021, 11:47 użytkownik Andrew Starovoyt <
***@***.***> napisał:

**Comment by Danon** [comment]
> Unfortunately, the problem is rather general, so if you see something similar please let us know.

From my perspective, it simply looked like MDX plugin simply searched for anything that looks like a JSX tag (e.g. `<Tag>` or `<column>`), **even** inside strings. So a fix would mean: look for JSX tags everywhere, but the strings.

**Comment by Danon** [comment]
@anstarovoyt Do you think the fix will be compatible with newest version of PhpStorm?

**Comment by anstarovoyt** [comment]
yes, it must be compatible with any IJ-based 2021.1 versions (PhpStorm, WebStorm, IJ IDEA, RubyMine, and so on)

**Comment by Danon** [comment]
@anstarovoyt The update didn't fix it all. For example this code, still breaks it

```jsx
<CodeTabs
  tregx="<c"/>
```

**Comment by Danon** [comment]
@anstarovoyt Remember that the attributes must work for all kinds of strings, so:

```jsx
<CodeTabs
  tregx="<c"/>
```

```jsx
<CodeTabs
  tregx={"<c"}/>
```

```jsx
<CodeTabs
  tregx={'<c'}/>
```

```jsx
<CodeTabs
  tregx={`<c`}/>
```

**Comment by anstarovoyt** [comment]
@Danon could you please share the full code snippet, because I cannot reproduce with problem with 1.2.211

**Comment by Danon** [comment]
Sure: https://github.com/T-Regx/T-Regx.github.io/blob/source/docs/match-group.mdx

**Comment by anstarovoyt** [comment]
the problem is different there (as I said before it is rather general). This sample doesn't work because of the multi-line attribute value and I am not sure that it can be easily handled in the current model.

**Comment by Danon** [comment]
@anstarovoyt So what can we do about it? Multiline attributes are valid in JSX.
